### ISSUE TITLE
Convertirlo en un repositorio general de problemas

### ISSUE THREAD

**Issue by mroman42** [dgiim/problemas#1]
Con problemas no sólo de seminarios, sino de todas las asignaturas del doble grado, así como problemas externos o que planteemos nosotros mismos.

**Comment by mroman42** [comment]
Quizá sería más fácil escribir enunciados y soluciones con markdown, en lugar de usar Latex, que es más difícil de usar para los textos breves que acaban constituyendo un problema o una solución.

**Comment by mx-psi** [comment]
Resumo lo que hemos estado hablando los de segundo:
- Debería ser para todos los cursos y estar dividido por asignaturas
- Tendríamos un foro (común y/o separado por cursos) donde poner dudas
- En el repositorio sólo se subirían los problemas una vez revisados
- Se podrían subir retos/posibles apuntes/ayudas pero no prácticas completas (o al menos no hasta que no haya pasado el plazo de entrega)

**Comment by josemam** [comment]
Vamos, que lo que estamos pensando en segundo tiene poco que ver con la naturaleza de este repo. (Va más orientado a ejercicios típicos que se proponen en los cursos)

**Comment by mroman42** [comment]
Sí, he entendido. Este repo puede contener tanto ejercicios típicos como de clase. Nos faltaría aquí el poder comentarlos bien (¿sistema de comentarios con MathJax?). Darle una cara un poco más accesible que un repo de github se puede hacer por encima, en la página del doble grado.

_PD: En Math.StackExchange usan ChatJax para los chats, pero no parece funcionar con los comentarios de GitHub._
